### PR TITLE
randomBetween

### PR DESCRIPTION
**Pull request by skreepatch**

### CODE REVIEW

File: utils/src/number/index.ts [hunk 1]
```diff
@@ -16,3 +16,14 @@ export function roundToDecimal(num, precision) {
     const factor = precision ? Math.pow(10, precision) : Math.pow(10, 2);
     return Math.round(num * factor) / factor;
 }
+
+
```
Comment (NetanelBasal) [inline utils/src/number/index.ts]: You should add tests inside the __tests__ folder.

Comment (NetanelBasal) [reply utils/src/number/index.ts]: Also, I think it's better to call it just `random` as lodash API.

Comment (NetanelBasal) [reply utils/src/number/index.ts]: Also, we now generate auto docs with `typedoc`, so please add description and  code example. (You can use markdown syntax).

Thanks!

Comment (skreepatch) [reply utils/src/number/index.ts]: Thanks, done!

File: utils/__tests__/number.spec.ts [hunk 2]
```diff
@@ -1,38 +1,72 @@
...
-            expect(roundToDecimal(3.14159265358979323846264, 17)).toEqual(3.14159265358979324);
+            expect(round(10, 5)).toEqual(10);
+            expect(round(17.17, 1)).toEqual(17.2);
+            expect(round(3.14159265358979323846264, 17)).toEqual(3.14159265358979324);
         });
     });
 
+    describe('random', () => {
+
+        it('should throw an error when 1st argument is not a number', () => {
```
Comment (NetanelBasal) [inline utils/__tests__/number.spec.ts]: 👏

**Pull request merged**